### PR TITLE
Some bug fix and implement some card view function

### PR DESCRIPTION
**Pull request by heartjigi**
1. on Edit CareerInfo, prevent save with blank answer.
 2. change SAVE -> Save
Card View
 1. build move on
 2. build shadow on android

### CODE REVIEW

File: src/components/UserList/CardScroll.js [hunk 1]
```diff
@@ -80,16 +80,9 @@ export default class CardScroll extends Component {
...
-    if (leftover > 0 && leftover < 100) {
-      // Swipe right(iOS doesn't work)
-      // this.resetListView.scrollTo({x: offsetX - leftover + interval});
-    } else if (leftover > (interval - 100)) {
-      // Swipe left(iOS doesn't work)
-      // this.resetListView.scrollTo({x: offsetX - leftover});
-    }
+    let index =  Math.round(event.contentOffset.x / interval);
+    if (index > this.props.dataSource._cachedRowCount - 1 ) index -= 1;
+    this.resetListView.scrollTo({x: (Math.round(event.contentOffset.x / interval)) * interval, y: 0, animated: true});
```
Comment (MechanicKim) [inline src/components/UserList/CardScroll.js]: @heartjigi  Please!
this.resetListView.scrollTo({
  x: (Math.round(event.contentOffset.x / interval)) * interval,
  y: 0,
  animated: true,
});

File: src/components/UserList/CardScroll.js [hunk 2]
```diff
@@ -135,8 +128,8 @@ export default class CardScroll extends Component {
     return (
       <ListView
         renderScrollComponent={this.renderScrollComponent}
+        onScrollEndDrag = {this.controlScroll.bind(this)}
```
Comment (MechanicKim) [inline src/components/UserList/CardScroll.js]: ' = ' or '=' ?

**Comment by MechanicKim** [comment]
I saw that some styles are in props of components and others are separated.
I think we need to organize styles!
Thanks! @heartjigi 

@SarahhhC @KyuaKwon @lyw0149  Could you guys review it?

File: src/components/UserList/CardScroll.js (hunk 1 above)
Comment (heartjigi) [reply src/components/UserList/CardScroll.js]: fixed! Thanks.

File: src/components/UserList/CardScroll.js (hunk 2 above)
Comment (heartjigi) [reply src/components/UserList/CardScroll.js]: fixed! Thanks.

**Comment by SarahhhC** [comment]
Good! LGTM 👍

**Pull request merged**